functests/4_latency/latency.go

```diff
@@ -40,6 +41,7 @@ const (
 	defaultTestRuntime  = "300"
 	defaultMaxLatency   = -1
 	defaultTestCpus     = -1
+	maxInt              = math.MaxInt32
```

Comment:
you do not need it under the new const, it already const and you can use it straight forward :)

Reply:
uh my mistake , correcting it , thanks !